### PR TITLE
MMP-504: ClassLoaderModel json file should not be written with pretty…

### PR DESCRIPTION
**Pull request by gsfernandes**
… print mode but users should be able to enable it by property

### CODE REVIEW

File: mule-classloader-model/src/main/java/org/mule/tools/api/classloader/ClassLoaderModelJsonSerializer.java [hunk 1]
```diff
@@ -52,8 +51,24 @@ public static ClassLoaderModel deserialize(File classLoaderModelDescriptor) {
...
+  }
+
+  /**
+   * Serializes the classloader model to a string
+   *
+   * @param classLoaderModel the classloader model of the application being packaged
+   * @param prettyPrinting if {@code true} the json will be printed with pretty print mode
+   * @return string containing the classloader model's JSON representation
+   */
+  private static String serialize(ClassLoaderModel classLoaderModel, boolean prettyPrinting) {
```
Comment (julianpascualMuleSoft) [inline mule-classloader-model/src/main/java/org/mule/tools/api/classloader/ClassLoaderModelJsonSerializer.java]: why it's private?

File: mule-packager/src/main/java/org/mule/tools/api/packager/sources/MuleContentGenerator.java [hunk 2]
```diff
@@ -106,10 +106,22 @@ public void createMetaInfMuleSourceFolderContent() throws IOException {
...
+    createApplicationClassLoaderModelJsonFile(classLoaderModel, true);
+  }
+
+  /**
+   * It creates classloader-model.json in META-INF/mule-artifact
+   *
+   * @param classLoaderModel the classloader model of the application being packaged
+   * @param prettyPrinting
+   */
+  @Deprecated
```
Comment (julianpascualMuleSoft) [inline mule-packager/src/main/java/org/mule/tools/api/packager/sources/MuleContentGenerator.java]: This new one shouldn't be marked as deprecated.

Comment (gsfernandes) [reply mule-packager/src/main/java/org/mule/tools/api/packager/sources/MuleContentGenerator.java]: good catch! I will fix it

File: mule-classloader-model/src/main/java/org/mule/tools/api/classloader/ClassLoaderModelJsonSerializer.java (hunk 1 above)
Comment (gsfernandes) [reply mule-classloader-model/src/main/java/org/mule/tools/api/classloader/ClassLoaderModelJsonSerializer.java]: I will make it public

**Pull request merged**